cpg_workflows/stages/large_cohort.py

```diff
@@ -31,7 +33,19 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
 
         j = get_batch().new_job('Combiner', (self.get_job_attrs() or {}) | {'tool': 'hail query'})
 
-        init_batch_args = {'worker_memory': 'highmem'} if get_config()['workflow'].get('highmem_workers') else None
+        init_batch_args = {}
```

Comment:
```suggestion
        init_batch_args: dict[str, str | int] = {}
```

suggesting this to make mypy happy